technical-decision-making-process/README.md

```diff
@@ -0,0 +1,84 @@
+---
+creation_date: 2024-05-22
+issues:
+- https://github.com/giantswarm/giantswarm/issues/30631
+owners:
+- https://github.com/orgs/giantswarm/teams/team-horizon
+state: review
+summary: This contains the process for proposing, discussing, and formalizing technical decisions. It also introduces the RFC structure.
+---
+
+# Structured way to propose, discuss, and formalize technical decisions within an organization
+
+### Problem statement
+
+The current technical decision-making process within our organization is inconsistent and inefficient, particularly when decisions impact multiple teams. This lack of a well-defined process leads to confusion, delays, and suboptimal decisions.
+We need a structured approach to proposing, discussing, and formalizing decisions to improve collaboration and decision quality.
+
+The first [RFC and decision making process](https://github.com/giantswarm/rfc/pull/78/files/) proposed a file structure and introduced an overview of RFCs in the [Handbook](https://handbook.giantswarm.io/docs/rfcs/).
```

Comment:
The RFC added in PR https://github.com/giantswarm/rfc/pull/78/ is now published as https://handbook.giantswarm.io/docs/rfcs/decision-process/. I would prefer using the final URL instead of the PR one.

Reply:
But then at least link the repo URL instead of the PR, right?
like https://github.com/giantswarm/rfc/blob/main/decision-process/README.md or are we worried we might change the process and want to make sure we link the original PR only?

Reply:
When we reference the rfc here we reference it to be read, or not? So if the best touchpoint to read it is the handbook, why not link that? 
But yes, at least if we want the repo to be the main touchpoint, we should use the repo url instead of the PR to not have to update this one whenever there is a change :)